ballerina/tests/52_unary_ldap_auth_client.bal

```diff
@@ -14,125 +14,140 @@
-    if response is Error {
-        test:assertEquals(response.message(), "Empty authentication header.");
-    } else {
-        test:assertFail(msg = "Expected grpc:Error not found.");
+    if !isWindowsEnvironment() {
+        HelloWorld52Client helloWorldEp = check new ("http://localhost:9152");
+        map<string|string[]> requestHeaders = {
+            "authorization": "Bearer "
+        };
+
+        ContextString requestMessage = {
+            content: "WSO2",
+            headers: requestHeaders
+        };
+        var response = helloWorldEp->testStringValueReturn(requestMessage);
+        if response is Error {
+            test:assertEquals(response.message(), "Empty authentication header.");
+        } else {
+            test:assertFail(msg = "Expected grpc:Error not found.");
+        }
     }
 }
 
 @test:Config {
-    enable: false,
+    enable: true,
     groups: ["ldap"]
 }
 isolated function testStringValueReturnWithUnauthorizedLdapAuth() returns Error? {
-    HelloWorld52Client helloWorldEp = check new ("http://localhost:9152");
-    map<string|string[]> requestHeaders = {};
-
-    CredentialsConfig config = {
-        username: "bob",
-        password: "bobgreen@123"
-    };
-
-    ClientBasicAuthHandler handler = new (config);
-    map<string|string[]>|ClientAuthError result = handler.enrich(requestHeaders);
-    if result is ClientAuthError {
-        test:assertFail(msg = "Test Failed! " + result.message());
-    } else {
-        requestHeaders = result;
-    }
+    if !isWindowsEnvironment() {
+        HelloWorld52Client helloWorldEp = check new ("http://localhost:9152");
+        map<string|string[]> requestHeaders = {};
+
+        CredentialsConfig config = {
+            username: "bob",
+            password: "bobgreen@123"
+        };
 
-    ContextString requestMessage = {
-        content: "WSO2",
-        headers: requestHeaders
-    };
-    var response = helloWorldEp->testStringValueReturn(requestMessage);
-    if response is Error {
-        test:assertEquals(response.message(), "Permission denied");
-    } else {
-        test:assertFail(msg = "Expected grpc:Error not found.");
+        ClientBasicAuthHandler handler = new (config);
+        map<string|string[]>|ClientAuthError result = handler.enrich(requestHeaders);
+        if result is ClientAuthError {
+            test:assertFail(msg = "Test Failed! " + result.message());
+        } else {
+            requestHeaders = result;
+        }
+
+        ContextString requestMessage = {
+            content: "WSO2",
+            headers: requestHeaders
+        };
+        var response = helloWorldEp->testStringValueReturn(requestMessage);
+        if response is Error {
+            test:assertEquals(response.message(), "Permission denied");
+        } else {
+            test:assertFail(msg = "Expected grpc:Error not found.");
+        }
     }
 }
+
+isolated function isWindowsEnvironment() returns boolean = @java:Method {
```

Comment:
Maybe we can put this function in `test_utils.bal`

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/356/commits/0680b45f1f9b83d53fa5a2c0a933c4dec92af6fb